src/plugins/intel_gpu/src/graph/impls/onednn/utils.cpp

```diff
@@ -345,6 +345,10 @@ cldnn::format find_data_format(dnnl::memory::desc desc) {
                     break;
                 }
             }
+            // b_fs_yx_fsv2 has dnnl::memory::format_tag::undef , so not select the format.
+            if (static_cast<format::type>(fmt_idx) == format::type::b_fs_yx_fsv2) {
+                is_match = false;
+            }
```

Comment:
Couldn't we remove the undef definition from table?  Let's discuss with Mingyu why it is added at the first by 'undef'.
And we might need test-case for this exception.

Reply:
As discussed, I updated to fallback to byxf when fsv2 is selected. Thanks!

Reply:
Proper place to implement such workaround would be `select_preferred_formats_for_onednn`, not in `find_data_format`.

Reply:
@isanghao Moved the WA code in select_preferred_formats_for_onednn. Thanks!